### PR TITLE
Some README and script suggestions.

### PR DESCRIPTION
**Pull request by edburns**
Author: Ed Burns <edward.burns@oracle.com>
Date:   Tue Aug 22 14:16:23 2017 -0400

    Merge branch 'master' of github.com:carimura/vista

Author: Ed Burns <edward.burns@oracle.com>
Date:   Fri Aug 18 19:51:31 2017 -0400

    Another small doc cleanup

    modified:   README.md

    - Mention the need for BUCKET to be oracle-vista-out

    modified:   scripts/setenv.sh

    - Hard code BUCKET to be oracle-vista-out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +43,11 @@ Ensure you have a GNU compatible make.
 1. `cd services; make deploy` (this should deploy all demo funcs to the Fn server) 
 1. set the proper ENV vars needed in scripts/setenv.sh, then run run `./setenv.sh`
 1. `open public/vista.html`
-1. Enter the value you're using for your BUCKET (default: oracle-vista-out) environment variable into the box (this subscribes to pubnub channel)
+1. Enter oracle-vista-out as the value of the BUCKET environment variable into the box (this subscribes to pubnub channel).
 
 ### Step 5: Local minio setup
 1. install the [mc minio client](https://github.com/minio/mc)
-1. Edit scripts/minio_config.json to change the webhook URL to the API_URL from step 4 above
+1. Edit scripts/minio_config.json to change the webhook URL to the API_URL from step 4 above  *Note, leave the `/r/myapp/publish` on the end!*  For example,
```
Comment (carimura) [inline README.md]: "For example," ... missing?

File: README.md [hunk 2]
```diff
@@ -5,7 +5,9 @@
 ## Running Locally
 
 This demo is designed to be run outside of a corporate firewall.  The
-moving parts have not been instrumented to run with proxies.
+moving parts have not been instrumented to run with proxies.  This demo
+does not work on the clear-guest WiFi because a key enabler of the demo
```
Comment (carimura) [inline README.md]: I'd prefer to keep Oracle-specific language out... My goal is to expose this demo wider than just Oracle.

File: README.md [hunk 3]
```diff
@@ -77,3 +79,181 @@ You should see activity in the ngrok logs, server logs, and output to the vista.
...
+boxes and the lines.  The boxes are the services and the lines are the
+wiring between the services.  Let's look at the wiring first.
+
+### Wiring
+
+The wiring for the demo uses a few enablers: ngrok and minio.
+
+ngrok is an insanely useful tool that essentially opens up a big
+security hole in your home router for as long as ngrok runs.  This is
+why the demo cannot run on clear-guest WiFi.  This is deemed an
```
Comment (carimura) [inline README.md]: Oracle reference

File: README.md [hunk 4]
```diff
@@ -77,3 +79,181 @@ You should see activity in the ngrok logs, server logs, and output to the vista.
...
+    PATCH /v1/apps/myapp                200 OK
+    PATCH /v1/apps/myapp                200 OK
+    PATCH /v1/apps/myapp                200 OK
+
+After this completes, you can do
+
+    fn routes list myapp
+    path  image     endpoint
+    /alert  edburns0docker/alert:0.1.18  f8cb781a.ngrok.io/r/myapp/alert
+    /detect-faces edburns0docker/detect-faces:0.1.12 f8cb781a.ngrok.io/r/myapp/detect-faces
```
Comment (carimura) [inline README.md]: might want to remove your Docker ID

File: README.md [hunk 5]
```diff
@@ -77,3 +79,181 @@ You should see activity in the ngrok logs, server logs, and output to the vista.
...
+    postURL := os.Getenv("FUNC_SERVER_URL") + "/draw"
+
+and also a copy of the same payload is posted to the alert service.
+
+    alertPostURL := os.Getenv("FUNC_SERVER_URL") + "/alert"
+
+#### Draw service
+
+We're back in Ruby for this one.  But the runtime of the func.yaml is ""
+instead of ruby.  Not sure why that is.  Some defaults thing?  This ruby
```
Comment (carimura) [inline README.md]: it's confusing I know. Fn [issue #231](https://github.com/fnproject/fn/issues/231) addresses this. Basically "" means use the Dockerfile to build the fun. This should change soon to "Docker".

File: README.md [hunk 6]
```diff
@@ -77,3 +79,181 @@ You should see activity in the ngrok logs, server logs, and output to the vista.
...
+service uses alpine-sdk and imagemagick.  Upon receipt of a payload,
+first we send a message to pubnub saying we are running an image.  The
+image is downloaded from the payload.  We use ImageMagick to draw the
+rectangles in the payload on the image.  We then upload the new image to
+MINIO, which looks like it is acting as a standin to Amazon S3.  Finally
+we send a message to pubnub saying this run is done.
+
+#### Alert service
+
+This one is in go again.  Runtime is the empty string.  Again, is this
```
Comment (carimura) [inline README.md]: "Go" capitalized

**Comment by edburns** [comment]
I have applied your changes in revision 1e4a8bb .

File: README.md [hunk 7]
```diff
@@ -77,3 +81,184 @@ You should see activity in the ngrok logs, server logs, and output to the vista.
...
+boxes and the lines. The boxes are the services and the lines are the
+wiring between the services. Let's look at the wiring first.
+
+### Wiring
+
+The wiring for the demo uses a few enablers: ngrok and minio.
+
+ngrok is an insanely useful tool that essentially opens up a big
+security hole in your home router for as long as ngrok runs. This is
+why the demo cannot run on a corporate guest WiFi. This is deemed an
```
Comment (carimura) [inline README.md]: actually I don't think we know whether ngrok will work in a particular corporate net/guest net ... Can you just say "May not work on Corp networks and guest wifi's"?

**Pull request merged**